pkg/app/piped/cloudprovider/lambda/function.go

```diff
@@ -164,3 +167,74 @@ func parseContainerImage(image string) (name, tag string) {
 	name = paths[len(paths)-1]
 	return
 }
+
+// FindArtifactVersions parses artifact versions from function.yaml.
+func FindArtifactVersions(fm FunctionManifest) ([]*model.ArtifactVersion, error) {
+	// Extract container image tag as application version.
+	if fm.Spec.ImageURI != "" {
+		name, tag := parseContainerImage(fm.Spec.ImageURI)
+		if name == "" {
+			return nil, fmt.Errorf("image name could not be empty")
+		}
+
+		return []*model.ArtifactVersion{
+			{
+				Kind:    model.ArtifactVersion_CONTAINER_IMAGE,
+				Version: tag,
+				Name:    name,
+				Url:     fm.Spec.ImageURI,
+			},
+		}, nil
+	}
+
+	// Extract s3 object version as application version.
+	if fm.Spec.S3ObjectVersion != "" {
+		return []*model.ArtifactVersion{
+			{
+				Kind:    model.ArtifactVersion_S3_OBJECT,
+				Version: fm.Spec.S3ObjectVersion,
+				Name:    fmt.Sprintf("%s/%s", fm.Spec.S3Bucket, fm.Spec.S3Key),
```

Comment:
@ffjlabo @khanhtc1202 
I feel that this name will be quite long. How about just showing the S3Key value?

Reply:
I see, certainly, it is long. Now it shows as "S3Bucket/S3Key: S3ObjectVersion" on Web UI and it's so long.
So I agree with that 🙌

Reply:
I'm okay with that 👌